### DISCUSSION TITLE
Capture on button press

### DISCUSSION THREAD

**Discussion by GLOUPY-RnD** [Lillifee/raspiCam#29] [Ideas]
Hi,
I really like your software !
It would be awesome to have the possibility to capture an image with a physical switch.
I'm looking for a way to do time lapses for my 3D printer but I can not use octoprint. It's a Raise3D Pro3 and the only solution for me is to have the head of the printer pressing a physical switch.
Anyway, keep it up !

**Comment by Lillifee** [comment]
done in version 2.0.0. https://github.com/Lillifee/raspiCam/issues/33
